html_app/assignments/scb_ex/addmutlpledialog.soy

```diff
@@ -1,93 +0,0 @@
-{namespace scb_ex_old}
-
-/**
-@param template
-@param assignment
-@param experiment
-*/
-{template .dialog}
-<div class='scb_ex_inner_dialog' role='dialog' aria-label='Add Samples'>
```

Comment:
Is there an easy way to view this dialog box somewhere? I'm assuming this was a duplicate of the `{template .dialog}` in the file above but there are minor differences